Yosemite/YosemiteTests/Mocks/CardPresentPayments/MockCardReader.swift

```diff
@@ -0,0 +1,23 @@
+@testable import Hardware
+
+struct MockCardReader {
+    static func bbpos() -> CardReader {
+        CardReader(serial: "WPE-SIMULATOR-1",
+                   vendorIdentifier: "SIMULATOR",
+                   name: "Simulated WisePOS E",
+                   status: .init(connected: false, remembered: false),
+                   softwareVersion: "0.0.0.1",
```

Comment:
Let's mimic the real version stings, e.g. `1.00.03.34-SZZZ_Generic_v45-300001` ( https://stripe.com/docs/terminal/readers/bbpos-chipper2xbt#reader-software-releases )

Reply:
Sure,  that being said, this is exactly the string that the Terminal SDK is providing when in test mode.